index.js

```diff
@@ -370,34 +370,33 @@ async function fastifyStatic (fastify, opts) {
       for (const [dirname, rootPath] of indexDirs.entries()) {
         const pathname = dirname + (dirname.endsWith('/') ? '' : '/')
         const file = '/' + pathname.replace(prefix, '')
-        setUpHeadAndGet(fastify, routeOpts, pathname, file, rootPath)
+        setUpHeadAndGet(routeOpts, pathname, file, rootPath)
 
         if (opts.redirect === true) {
-          setUpHeadAndGet(fastify, routeOpts, pathname.replace(/\/$/, ''), file.replace(/\/$/, ''), rootPath)
+          setUpHeadAndGet(routeOpts, pathname.replace(/\/$/, ''), file.replace(/\/$/, ''), rootPath)
         }
       }
     }
   }
 
-  function setUpHeadAndGet (fastify, routeOpts, route, file, rootPath) {
+  function setUpHeadAndGet (routeOpts, route, file, rootPath) {
     const toSetUp = {
       ...routeOpts,
       method: ['HEAD', 'GET'],
       url: route,
-      handler: serveFileHandler
+      handler (req, reply) {
```

Comment:
```suggestion
      handler: function serveFileHandler (req, reply) {
```

It changes the name in case of https://github.com/Eomm/fastify-overview/

Reply:
Damn fastify-overview looks cool, will revert the change